### PR TITLE
add examples in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jaygala223**
Fixed spellings and added some examples in CONTRIBUTING.md

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+
+### Examples
+
+#### Branch Names: `{branch_type}/{issue-tracker-id-}issue-one-liner`
+1. feature/2234-infinite-scroll
+2. documentation/3344-linux-installation
+3. test/2222-unit-tests
+
+#### Commits
+1. [Closes #1234] Submit button added
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
1. [#1234] feature: Submit button added.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -2,10 +2,22 @@
...
+2. [Fixes #1232] Infinite scroll fixed
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
2. [#1232] fix: Infinite scroll fixed
```

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -2,10 +2,22 @@
...
+3. Add unit tests for xyz feature
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md:53]:
```suggestion
3. [#333] test: Add unit tests for xyz feature
```

**Review by sudiptob2 — COMMENTED** [review]
Thanks, suggested some change

**Comment by sudiptob2** [comment]
This is the GitHub Workflow I use and want others to use while contributing to this project. For example, I am a contributor and you are the maintainer of this project. I will take the following steps while contributing to the project.
1. I have a ticket
2. I make small changes on a new branch, then make a PR. the PR is draft
3. then do other related works and push to the remote branch
4. meanwhile, you can review if you like, and put comments. and I'll answer or update you. this way I get early review or share code for decision, etc
5. now my work is done, I changed PR status from draft to active (ready for review) -> meaning my part is done, what do other people think of it
6. You approve the PR.
7. I'll check if the base branch is updated, If not update and sync my branch.
8. Maintainer hit the big merge button.

I want to add a section describing these steps. can you please update the readme with these changes? :pray:  

**Note: I have written these steps plainly, please feel free to do modifications regarding grammar, lexical resources, etc to make it more professional.**

**Comment by jaygala223** [comment]
Sure. I'll update the readme with these changes. Thanks

**Comment by sudiptob2** [comment]
> Sure. I'll update the readme with these changes. Thanks

Awesome :100:

File: README.md [hunk 4]
```diff
@@ -44,7 +44,35 @@ crontab -e
...
-## Contribution guideline
+## Contributing
 
-If you like this project and want to improve by adding features, fixing bugs or anything, please follow
-the [contributing guidelines](docs/CONTRIBUTING.md).
\ No newline at end of file
+We are very happy to see you here. Before sending your pull requests, make sure that you read the whole workflow and the contributing guidelines given below. If you have any doubt regarding the contributing guide, please feel free to [state it clearly in an issue](https://github.com/sudiptob2/leet-easy/issues/new/choose)
+
+
+### Workflow
```
Comment (sudiptob2) [inline README.md]: I think this workflow part should be moved into `contributing.md`. Can you update ?

**Review by sudiptob2 — COMMENTED** [review]
Seems great :rocket: I suggested some small changes :blush:

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -1,15 +1,47 @@
...
+
+2. Fork the project
+
+3. Create a branch and make small changes on it.
+
+4. Create a **draft PR**
+
+5. Then make other changes and push to the remote branch you created.
+
+In this way, the maintainers will be able to provide early reviews and comments for your commits which will save time later on.
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
```

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -1,15 +1,47 @@
...
+
+    1.1. Clarify any doubts in the comments section of the issue.
+
+2. Fork the project
+
+3. Create a branch and make small changes on it.
+
+4. Create a **draft PR**
+
+5. Then make other changes and push to the remote branch you created.
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
5. Then make other changes and push to the remote branch you created. In this way, the maintainers will be able to provide early reviews and comments for your commits which will save time later on.
```
```suggestion
5. Then make other changes and push to the remote branch you created.
```

File: docs/CONTRIBUTING.md [hunk 7]
```diff
@@ -1,15 +1,47 @@
+Given below is the general workflow we expect you to follow while making contributions to this project.
+
+<br>
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
```

File: docs/CONTRIBUTING.md [hunk 8]
```diff
@@ -1,15 +1,47 @@
...
+
+3. Create a branch and make small changes on it.
+
+4. Create a **draft PR**
+
+5. Then make other changes and push to the remote branch you created.
+
+In this way, the maintainers will be able to provide early reviews and comments for your commits which will save time later on.
+
+6. Once the above steps are done, you can change the PR status from *draft to active*
```
Comment (sudiptob2) [inline docs/CONTRIBUTING.md]:
```suggestion
6. Once the above steps are done, you can change the PR status from **draft to active**
```

**Comment by jaygala223** [comment]
Thanks for the suggestions. I have made those changes and committed

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (sudiptob2) [reply docs/CONTRIBUTING.md:53]: @jaygala223 can you update this? :pray:

**Review by sudiptob2 — APPROVED** [review]
One suggestion missed, however I am approving this PR

**Comment by jaygala223** [comment]
Umm sorry. What did I miss?

**Comment by sudiptob2** [comment]
> Umm sorry. What did I miss?

[here](https://github.com/sudiptob2/leet-easy/pull/45#discussion_r989338600)

File: docs/CONTRIBUTING.md (hunk 3 above)
Comment (jaygala223) [reply docs/CONTRIBUTING.md:53]: Yes. I'll do it.
Should I make a new PR or this one can be used?

Comment (sudiptob2) [reply docs/CONTRIBUTING.md:53]: make a new pr please

Comment (jaygala223) [reply docs/CONTRIBUTING.md:53]: Okay

**Pull request merged**